rest/models/recruitment.py

```diff
@@ -64,3 +65,38 @@ def to_json(rec, candidates_num):
             'major_name': rec.major.name,
             'major_mode': str(rec.major.mode)
         }
+
+    @staticmethod
+    def get_cycles_summary(recruitments):
+        """
+        Return json with summary of recruitments
+        :param recruitments: list of recruitments
+        :return: summary dict
+        """
+        rec = recruitments[0]
+        candidates_number = 0
+        overall_qualified = 0
+        min_point_limit = None
+        is_active = False
+        for recruitment in recruitments:
+            candidates_number += len(recruitment.candidate_recruitments)
+            if recruitment.point_limit is None:
+                is_active = True
+            else:
+                overall_qualified += sum(1 for candidate in recruitment.candidate_recruitments
+                                         if candidate.status == RecruitmentStatus.QUALIFIED)
+                if min_point_limit is not None:
+                    min_point_limit = min(min_point_limit, recruitment.point_limit)
+                else:
+                    min_point_limit = recruitment.point_limit
```

Comment:
min_point_limit is always >= 0 so maybe init it with -1 and check if min_point_limit >= 0